spec/music_album_spec.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'spec_helper'
+require 'date'
+
+describe 'music_album' do
+  before(:each) do
+    @music_album = MusicAlbum.new('Salsa', 'Ruben Blades', 'Classic', Date.new(6 - 20 - 1983))
```

Comment:
Great job so far, it's almost complete!

Please refactor the MusicAlbum class and the test so it only takes the `publish_date and the on_spotify` properties
(ex: `MusicAlbum(on_spotify, publish_Date`)

**OPTIONAL:**
Also, in tests try no to use magic strings, create variables to hold the values and then pass it to where you need them (ex: `on_spotify = true` and `MusicAlbum.new(on_spotify)`)